build/docker-compose.yaml

```diff
@@ -55,6 +54,17 @@ services:
     depends_on:
       - coordinator
 
+  oneshot:
```

Comment:
As oneshot is a submission client, this shouldn't be in the compose file, right?

Reply:
You're probably right that it should have its own compose file, I'll make that change.